Makefile

```diff
@@ -29,6 +29,16 @@ test-minimal:  ## Launch minimal test suite
 	mkdir -p tests/logs
 	ANSIBLE_CONFIG=$(TEST_CONFIG) ansible-playbook -v -i $(TEST_INVENTORY) -e @$(TEST_VARS) -e @$(TEST_SECRETS) $(TEST_ARGS) tests/playbooks/test_minimal.yaml 2>&1 | tee $(TEST_OUTFILE)
 
+test_ospdo_pre_dataplane: TEST_OUTFILE := tests/logs/test_minimal_out_$(shell date +%FT%T%Z).log
```

Comment:
Could also be called 'test_ospdo_controlplane' but i'm just nitpicking. It's good to merge IMO.